apps/test/unit/code-studio/pd/application/TeacherApplicationIntegrationTest.js

```diff
@@ -55,13 +59,15 @@ describe('TeacherApplication', () => {
     sinon.assert.calledThrice(firehoseClient.putRecord);
   });
   it('Does not set schoolId if not provided', () => {
-    const page = mount(<TeacherApplication {...defaultProps} />);
+    const page = mount(<FindYourRegion {...defaultProps} />);
     expect(page.find('SchoolAutocompleteDropdown').prop('value')).to.equal(
       undefined
     );
   });
```

Comment:
if you're making any other changes, can you add new lines between these tests? I don't want you to go through another CI cycle for it as this is an existing issue but it's kind of hard to read where one test ends and the next begins without them

Reply:
On it!